amgut/lib/data_access/sql_connection.py

```diff
@@ -53,6 +53,7 @@ def __init__(self):
         self._connection = None
         self._post_commit_funcs = []
         self._post_rollback_funcs = []
+        self.auto_rollback = False
```

Comment:
That would add 45 seconds to a minute per test, and was agreed upon with @wasade when we first started adding tests would not be done.